### PR TITLE
Update repo structure

### PR DESCRIPTION
**Pull request by rudolfs**
Most controversial change of 2020.

Refs #130

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ Here are a couple of examples:
...
 
 The pull request ref goes in brackets at the end of the subject.
-Generally we also follow [good commit message hygene][tpope].
+Generally we also follow [good commit message hygene][tp].
 
 
 ## Work coordination
 
-Work on the app and proxy can happen in parallel. However for this to work the
+Work on the UI and proxy can happen in parallel. However for this to work the
```
Comment (xla) [inline DEVELOPMENT.md]:
```suggestion
Work on the UI and proxy can happen in parallel. However for this to work, the
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -54,51 +54,371 @@ Annotations for queries, mutations, types and fields also get propagated from
...
+
+1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`.
+2. Install dependencies: `cd radicle-upstream && yarn install`.
+3. Start the app in development mode: `yarn start`.
+
+
+### Running tests
+
+Before running tests locally you'll need to set up a test fixture repository:
+`git submodule foreach "git fetch --all"`.
```
Comment (xla) [inline DEVELOPMENT.md]: Does this work without a `git submodule update --init --remote` before?

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -54,51 +54,371 @@ Annotations for queries, mutations, types and fields also get propagated from
...
+
+- to run tests do: `yarn test`
```
Comment (xla) [inline DEVELOPMENT.md]:
```suggestion
- to run tests call: `yarn test`
```

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -54,51 +54,371 @@ Annotations for queries, mutations, types and fields also get propagated from
...
+    ├── notification.js
+    ├── project.js
+    └── sourceBrowser.js
+```
 
-  👉 https://github.com/rudolfs/test/pull/17
+#### State
 
-To merge the pull request and finalize this release run:
+State shared across multiple component fragments which together build a more
```
Comment (xla) [inline DEVELOPMENT.md]:
```suggestion
State shared across multiple Screen fragments which together build a more
```

File: DEVELOPMENT.md [hunk 5]
```diff
@@ -153,11 +473,62 @@ DOCKER_IMAGE: 'gcr.io/opensourcecoin/radicle-upstream:0.2.1'
    should build the new branch with the updated image.
 
 
-[tpope]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+## Releases
+
+To perform a release:
+
+1. If you haven't already, install the [`hub`][hb] cli tool.
```
Comment (xla) [inline DEVELOPMENT.md]: Maybe expand what in our release process depends on hub.

File: DEVELOPMENT.md (hunk 2 above)
Comment (rudolfs) [reply DEVELOPMENT.md]: Nope, we need to init submodules before. Will add.

File: README.md [hunk 6]
```diff
@@ -5,28 +5,16 @@
...
-a good way to explore the project is to read the documentation and have a go at
-building it locally.
-
-A good entry-point for exploration is [`app/README.md`][re], where you'll find
+At the moment we support Linux and macOS. Later on we'll provide ready-made
+packages for both platforms, however for now a good way to explore the project
+is to read the documentation and have a go at building it locally.
+
+The [UI][ui] is written in JavaScript using Svelte and Electron and the node
+[proxy][pr] logic is implemented in rust.
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply README.md]: Good point!

File: DEVELOPMENT.md (hunk 2 above)
Comment (rudolfs) [reply DEVELOPMENT.md]: It does not.

**Pull request merged**